connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourceModelNodeUtil.java

```diff
@@ -187,7 +186,7 @@ static ModifiableDataSource from(final OperationContext operationContext, final
         final DsSecurity security = new DsSecurityImpl(username, password,
                 elytronEnabled? authenticationContext: securityDomain, elytronEnabled, credentialSourceSupplier, reauthPlugin);
 
-        final boolean sharePreparedStatements = ModelNodeUtil.getBooleanIfSetOrGetDefault(operationContext, dataSourceNode, SHARE_PREPARED_STATEMENTS);
+        final Boolean sharePreparedStatements = ModelNodeUtil.getBooleanIfSetOrGetDefault(operationContext, dataSourceNode, SHARE_PREPARED_STATEMENTS);
```

Comment:
This should never be null as SHARED_PREPARED_STATEMENTS has a default value. ModelNodeUtil.getBooleanIfSetOrGetDefault can return null, but not when this value is passed in.

Reply:
Yeah, I wanted to get rid of IDE inspection warning...